src/model/annotation-token.v1.yaml

```diff
@@ -0,0 +1,12 @@
+$schema: http://json-schema.org/draft-04/schema#
+title: Annotation token
+type: object
+allOf:
+  - properties:
+        token:
+            $ref: ../misc/jwt.v1.yaml
+        expire:
```

Comment:
we've seen in `profiles` `expires_at` to distuinguish between moments in time and intervals (which would be `expires_in`). Perhaps here too?

Reply:
Will change to `expiresAt`